doc/source/howtos/virt_sandbox.rst

```diff
@@ -141,169 +102,259 @@ with application servers.
+
+   .. warning::
+
+      Do not use default credentials in production!
+
+   The following example shows how to connect to the ProxySQL admin interface
+   with default credentials:
+
+   .. code-block:: bash
+
+      root@proxysql:~# mysql -u admin -padmin -h 127.0.0.1 -P 6032
+
+      Welcome to the MySQL monitor.  Commands end with ; or \g.
+      Your MySQL connection id is 2
+      Server version: 5.5.30 (ProxySQL Admin Module)
+
+      Copyright (c) 2009-2020 Percona LLC and/or its affiliates
+      Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+
+      Oracle is a registered trademark of Oracle Corporation and/or its
+      affiliates. Other names may be trademarks of their respective
+      owners.
+
+      Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+      mysql>
+
+   To see the ProxySQL databases and tables use the following commands:
+
+   .. code-block:: text
+
+      mysql> SHOW DATABASES;
+      +-----+---------+-------------------------------+
+      | seq | name    | file                          |
+      +-----+---------+-------------------------------+
+      | 0   | main    |                               |
+      | 2   | disk    | /var/lib/proxysql/proxysql.db |
+      | 3   | stats   |                               |
+      | 4   | monitor |                               |
+      +-----+---------+-------------------------------+
+      4 rows in set (0.00 sec)
+
+   .. code-block:: text
+
+      mysql> SHOW TABLES;
+      +--------------------------------------+
+      | tables                               |
+      +--------------------------------------+
+      | global_variables                     |
+      | mysql_collations                     |
+      | mysql_query_rules                    |
+      | mysql_replication_hostgroups         |
+      | mysql_servers                        |
+      | mysql_users                          |
+      | runtime_global_variables             |
+      | runtime_mysql_query_rules            |
+      | runtime_mysql_replication_hostgroups |
+      | runtime_mysql_servers                |
+      | runtime_scheduler                    |
+      | scheduler                            |
+      +--------------------------------------+
+      12 rows in set (0.00 sec)
+
+   For more information about admin databases and tables,
+   see `Admin Tables
+   <https://github.com/sysown/proxysql/blob/master/doc/admin_tables.md>`_
+
+   .. note::
+
+      ProxySQL has 3 areas where the configuration can reside:
+
+      * MEMORY (your current working place)
+      * RUNTIME (the production settings)
+      * DISK (durable configuration, saved inside an SQLITE database)
+
+      When you change a parameter, you change it in MEMORY area.
+      That is done by design to allow you to test the changes
+      before pushing to production (RUNTIME), or save them to disk.
+
+   .. rubric:: Adding cluster nodes to ProxySQL
```

Comment:
ProxySQL2 natively supports PXC. It is handling through table [runtime_]mysql_galera_hostgroups. Use this blog post to get the updated SQLs for ProxySQL2-PXC setup: https://www.percona.com/blog/2019/02/20/proxysql-native-support-for-percona-xtradb-cluster-pxc/

Reply:
Tracked in another issue PXC-3251